bedrock/mozorg/templates/mozorg/contribute/index.html

```diff
@@ -150,6 +150,23 @@ <h2 class="section-title">{{ _('Not ready to dive in just yet?') }}</h2>
 
 {% endblock contrib_content %}
 
+{% block cta %}
+
+  {% set localized_surveys = {'en-US': 'GHJYW6D'} %}
+
+  {% if LANG in localized_surveys %}
+    <div id="survey-message">
+      <span class="survey-invite">
+        {{ _('Hello! Would you be willing to take a minute to answer a few questions?') }}
+      </span>
+      <a class="survey-button" href="https://www.surveymonkey.com/r/{{ localized_surveys[LANG] }}">
+        {{ _('Sure. I’ll help.') }}
+      </a>
+    </div>
+  {% endif %}
+
+{% endblock %}
+
 {% block js %}
   {{ js_bundle('contribute-base') }}
   {{ js_bundle('contribute-landing') }}
```

Comment:
Suggest adding a bundle just for the survey here, which we can turn on / off via a switch:

```
  {% if switch('contribute-survey') %}
    {{ js_bundle('contribute-survey') }}
  {% endif %}
```

Reply:
Since the feature doesn't rely only on JS to be toggled I added a switch to the `div` as well.